task/common/machine/storage.go

```diff
@@ -132,7 +132,7 @@ func Transfer(ctx context.Context, source, destination string, include string) e
 		return err
 	}
 
-	defer progress(2 * time.Second)()
+	defer progress(10 * time.Second)()
```

Comment:
Why 10? The transfer log might be also synced here? Is there any chance that the logs might be repeated?

Reply:
It's just to reduce noise

Reply:
Instead of updating the progress every 2 seconds, it updates the progress every 10 seconds.

Reply:
See https://github.com/iterative/terraform-provider-iterative/pull/467#discussion_r839871754 for background.

Reply:
- can we make it user-configurable somehow?
- can we merge it with the "still creating/destroying" messages?

Reply:
> _can we merge it with the "still creating/destroying" messages?_

Yes, forking Terraform Core or with `LD_PRELOAD=do_this.so` 🙃

Reply:
> _can we make it user-configurable somehow?_

Yes, either as part of the schema or through environment variables, as per https://github.com/iterative/terraform-provider-iterative/pull/467#discussion_r839961885. I wonder if this kind of parameters are important enough to make them configurable.

Reply:
✅ Merged